README.md

```diff
@@ -195,6 +227,21 @@ I, [2023-11-23T22:41:39.148523 #1]  DEBUG -- : [my-own-prefix] Request has been
 I, [2023-11-23T22:41:39.148537 #1]  DEBUG -- : [my-own-prefix] Returning the response from the original request.
 ```
 
+### manage_redis_exceptions
+
+By default, the `grape-idempotency` gem is configured to re-raise `Redis` exceptions.
+
+If you want to delegate the `Redis` exception management into the gem, you can configure it using the `manage_redis_exceptions` configuration property.
+
+```ruby
+Grape::Idempotency.configure do |c|
+  c.storage = @storage
+  c.manage_redis_exceptions = true
```

Comment:
Should this be called 'manage_storage_exceptions' instead? As the storage could be 'any' and not just redis

Reply:
Oks, I see in the README at the beginning it says redis instance should be passed as storage, and also redis is a runtime dependency. Just got confused by the method `valid_storage?`

Reply:
I don't think so, as we are rescuing explicitly `Redis::BaseError` 

https://github.com/jcagarcia/grape-idempotency/blob/c15b2e08915c3d9a4ea3d49bcdf56b44a0a020e2/lib/grape/idempotency.rb#L162

Also, in our documentation we are talking always about `Redis` storage.

https://github.com/jcagarcia/grape-idempotency/blob/c15b2e08915c3d9a4ea3d49bcdf56b44a0a020e2/README.md?plain=1#L35-L45

However, it's true that any class that implements `get`, `set`, `del` should work as an storage.

https://github.com/jcagarcia/grape-idempotency/blob/c15b2e08915c3d9a4ea3d49bcdf56b44a0a020e2/lib/grape/idempotency.rb#L134-L138

What do you think?

Reply:
Maybe for now, in this version we can make the dependency with Redis explicit.
Then, in future versions it can be refactored to support other storages